hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3InitiateMultipartUploadRequest.java

```diff
@@ -118,6 +127,28 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
       validateBucketAndVolume(omMetadataManager, volumeName, bucketName);
 
+      // Check if this transaction is a replay.
+      // We check only the KeyTable here and not the OpenKeyTable. In case
+      // this transaction is a replay but the transaction was not committed
+      // to the KeyTable, then we recreate the key in OpenKey table. This is
+      // okay as all the subsequent transactions would also be replayed and
+      // the openKey table would eventually reach the same state.
+      // The reason we do not check the OpenKey table is to avoid a DB read
+      // in regular non-replay scenario.
+      String dbKeyName = omMetadataManager.getOzoneKey(volumeName, bucketName,
```

Comment:
Looks like checking keyTable is not required here. I feel we can ignore it here.